### PR TITLE
zh, en: optimize access-dashboard.md

### PR DESCRIPTION
**Pull request by TomShawn**
### What is changed, added, or deleted? (Required)
Optimize access-dashboard.md

### Which TiDB Operator version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v1.2 (TiDB Operator 1.2 versions)
- [ ] v1.1 (TiDB Operator 1.1 versions)
- [ ] v1.0 (TiDB Operator 1.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: 
- Other reference link(s): 

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: zh/access-dashboard.md [hunk 1]
```diff
@@ -30,13 +37,13 @@ spec:
...
-## 通过端口转发访问 TiDB Dashboard
+## 方法 1. 通过端口转发访问 TiDB Dashboard
 
-> **注意：**
+> **警告：**
 >
 > 以下教程仅为演示如何快速访问 TiDB Dashboard，请勿在生产环境中直接使用以下方法。
 
-`TiDB Dashboard` 目前在 4.0.0 版本及以上中已经内嵌在了 PD 组件中，你可以通过以下的例子在 Kubernetes 环境下快速部署一个 4.0.4 版本的 TiDB 集群。运行 `kubectl apply -f` 命令，将以下 yaml 文件部署到 Kubernetes 集群中。
+`TiDB Dashboard` 目前在 4.0.0 版本及以上中已经内嵌在了 PD 组件中，你可以参考以下示例在 Kubernetes 环境下快速部署一个 4.0.4 版本的 TiDB 集群。运行 `kubectl apply -f` 命令，将以下 yaml 文件部署到 Kubernetes 集群中。
```
Comment (DanielZhangQD) [inline zh/access-dashboard.md:46]:
```suggestion
`TiDB Dashboard` 目前在 4.0.0 版本及以上中已经内嵌在了 PD 组件中，你可以参考以下示例在 Kubernetes 环境下快速部署一个 TiDB 集群。运行 `kubectl apply -f` 命令，将以下 yaml 文件部署到 Kubernetes 集群中。
```

File: zh/access-dashboard.md [hunk 2]
```diff
@@ -185,7 +192,7 @@ spec:
 
 需要注意如果 PD Pod 数量超过 1 ，需要在 TidbCluster CR 中设置 `spec.pd.enableDashboardInternalProxy: true` 以保证正常访问 TiDB Dashboard。
 
-## 更新 TiDB 集群
+## 方法 4. 更新 TiDB 集群
```
Comment (DanielZhangQD) [inline zh/access-dashboard.md:195]: 这块删了就行，前置条件里面有了

Comment (TomShawn) [reply zh/access-dashboard.md]: Done

**Review by DanielZhangQD — APPROVED** [review]
LGTM

File: zh/access-dashboard.md [hunk 3]
```diff
@@ -4,21 +4,27 @@ summary: 介绍如何在 Kubernetes 环境下访问 TiDB Dashboard
...
 
-TiDB Dashboard 是从 TiDB 4.0 开始引入的专门用来帮助观察与诊断整个 TiDB 集群的可视化面板，你可以在 [TiDB Dashboard](https://docs.pingcap.com/zh/tidb/stable/dashboard-intro) 了解详情。本篇文章将介绍如何在 Kubernetes 环境下访问 TiDB Dashboard。
+- 在测试环境中，你可以[通过端口转发访问 TiDB Dashboard](#方法-1-通过端口转发访问-tidb-dashboard)。
+- 在生产环境中，推荐[通过 Ingress 访问 TiDB Dashboard](#方法-2-通过-ingress-访问-tidb-dashboard)，并选择开启 TLS 加密传输，见[使用 Ingress 并开启 TLS](#使用-ingress-并开启-tls)。
+- 如果需要使用非域名的方式访问 TiDB Dashboard，可以选择[使用 NodePort Service](#方法-3-使用-nodeport-service)。
 
 > **注意：**
 >
-> TiDB Operator 会为每一个 TiDB 集群启动一个 Discovery 服务。Discovery 服务会为每个 PD Pod 返回相应的启动参数，来辅助 PD 集群启动。此外，Discovery 服务也会发送代理请求到 TiDB Dashboard。本文档我们将通过 Discovery 服务访问 TiDB Dashboard。
+> TiDB Dashboard 中的部分功能会因为 kubernetes 的特殊环境而无法使用，详见 [TiDB Operator 中不支持的 Dashboard 功能](#tidb-operator-中不支持的-dashboard-功能)。
```
Comment (hfxsd) [inline zh/access-dashboard.md:17]:
```suggestion
> TiDB Dashboard 中的部分功能会因为 Kubernetes 的特殊环境而无法使用，详见 [TiDB Operator 中不支持的 Dashboard 功能](#tidb-operator-中不支持的-dashboard-功能)。
```

File: zh/access-dashboard.md [hunk 4]
```diff
@@ -161,9 +167,9 @@ type: kubernetes.io/tls
 
 当 Ingress 部署完成以后，你就可以通过 <https://{host}/dashboard> 访问 TiDB Dashboard。
 
-### 使用 NodePort Service
+## 方法 3. 使用 NodePort Service
 
-由于 `Ingress` 必需使用域名访问，在某些场景下可能难以使用，此时可以通过添加一个 `NodePort` 类型的 `Service` 来访问和使用 `TiDB Dashboard`.
+由于 `Ingress` 必需使用域名访问，在某些场景下可能难以使用，此时可以通过添加一个 `NodePort` 类型的 `Service` 来访问和使用 `TiDB Dashboard`。
 
 以下是一个使用 `NodePort` 类型的 `Service` 访问 `TiDB Dashboard` 的 yaml 文件例子。运行 `kubectl apply -f` 命令，将以下 yaml 文件部署到 Kubernetes 集群中。
```
Comment (hfxsd) [inline zh/access-dashboard.md:173]: 开发在review我的快速上手的时候，把 `kubectl apply` 命令都改成 `kubectl create` 了，请和开发确认下这里是否也要改。

File: zh/access-dashboard.md [hunk 5]
```diff
@@ -190,27 +196,12 @@ spec:
...
 TiDB Dashboard 中的部分功能会因为 kubernetes 的特殊环境而无法使用，包括以下功能：
 
-1. **概况** -> **监控和告警** -> **查看监控项**的链接无法正确跳转到 Grafana 监控页。如果需要访问 Grafana 监控，可以参考[访问 Grafana 监控面板](monitor-a-tidb-cluster.md#访问-grafana-监控面板)。
++ **概况** -> **监控和告警** -> **查看监控项**的链接无法正确跳转到 Grafana 监控页。如果需要访问 Grafana 监控，可以参考[访问 Grafana 监控面板](monitor-a-tidb-cluster.md#访问-grafana-监控面板)。
 
-2. 日志搜索功能无法使用。如果需要查看对应组件的日志，可以使用 `kubectl logs ${pod_name} -n {namespace}` 查看对应组件的日志或者通过 Kubernetes 集群的日志服务查看。
++ 日志搜索功能无法使用。如果需要查看对应组件的日志，可以使用 `kubectl logs ${pod_name} -n {namespace}` 查看对应组件的日志或者通过 Kubernetes 集群的日志服务查看。
 
-3. **集群信息** -> **主机** 的磁盘容量，磁盘使用率无法正确显示。可以通过 [TidbMonitor 监控面板](monitor-a-tidb-cluster.md#访问-grafana-监控面板) 的各组件 dashboards 查看各个组件的磁盘使用或者部署 [Kubernetes 宿主机 Grafana 监控](monitor-kubernetes.md#kubernetes-组件监控) 查看 Kubernetes 节点的磁盘使用情况。
++ **集群信息** -> **主机** 的磁盘容量，磁盘使用率无法正确显示。可以通过 [TidbMonitor 监控面板](monitor-a-tidb-cluster.md#访问-grafana-监控面板) 的各组件 dashboards 查看各个组件的磁盘使用或者部署 [Kubernetes 宿主机 Grafana 监控](monitor-kubernetes.md#kubernetes-组件监控) 查看 Kubernetes 节点的磁盘使用情况。
```
Comment (hfxsd) [inline zh/access-dashboard.md:206]:
```suggestion
+ **集群信息** -> **主机** 的磁盘容量，磁盘使用率无法正确显示。可以通过 [TidbMonitor 监控面板](monitor-a-tidb-cluster.md#访问-grafana-监控面板) 的各组件 dashboards 查看各个组件的磁盘使用，或者部署 [Kubernetes 宿主机 Grafana 监控](monitor-kubernetes.md#kubernetes-组件监控) 查看 Kubernetes 节点的磁盘使用情况。
```

**Review by hfxsd — APPROVED** [review]
Rest LGTM

File: zh/access-dashboard.md (hunk 4 above)
Comment (TomShawn) [reply zh/access-dashboard.md:173]: @DanielZhangQD PTAL

Comment (DanielZhangQD) [reply zh/access-dashboard.md:172]: 这里没关系，都行

File: en/access-dashboard.md [hunk 6]
```diff
@@ -6,17 +6,23 @@ aliases: ['/docs/tidb-in-kubernetes/dev/access-dashboard/']
...
+- To access TiDB Dashboard without a domain name, you can [use NodePort Service](#method-3-use-nodeport-service).
+
 > **Note:**
 >
-> TiDB Operator starts a Discovery service for each TiDB cluster. The Discovery service can return the corresponding startup parameters for each PD Pod to support the startup of the PD cluster. The Discovery service can also send proxy requests to the TiDB Dashboard. In this document, TiDB Dashboard is accessed using the Discovery service.
+> Due to the special environment of Kubernetes, some features of TiDB Dashboard are not supported in TiDB Operator. See [Unsupported TiDB Dashboard features](#unsupported-tidb-dashboard-features) for details.
+
+In this document, TiDB Dashboard is accessed using the Discovery service. TiDB Operator starts a Discovery service for each TiDB cluster. The Discovery service can return the corresponding startup parameters for each PD Pod to support the startup of the PD cluster. The Discovery service can also send proxy requests to the TiDB Dashboard.
```
Comment (hfxsd) [inline en/access-dashboard.md:21]:
```suggestion
In this document, you can access TiDB Dashboard using the Discovery service. TiDB Operator starts a Discovery service for each TiDB cluster. The Discovery service can return the corresponding startup parameters for each PD Pod to support the startup of the PD cluster. The Discovery service can also send proxy requests to the TiDB Dashboard.
```

File: en/access-dashboard.md [hunk 7]
```diff
@@ -34,15 +40,15 @@ spec:
...
 
 > **Warning:**
 >
-> This guide shows how to quickly access TiDB Dashboard. Do **NOT** use this method in the production environment. For production environments, refer to [Access TiDB Dashboard by Ingress](#access-tidb-dashboard-by-ingress).
+> This guide shows how to quickly access TiDB Dashboard. Do **NOT** use this method in the production environment. For production environments, refer to [Access TiDB Dashboard by Ingress](#method-2-access-tidb-dashboard-by-ingress).
 
 TiDB Dashboard is built in the PD component in TiDB 4.0 and later versions. You can refer to the following example to quickly deploy a TiDB cluster in Kubernetes.
 
-1. Deploy the following `.yaml` file into the Kubernetes cluster by running the `kubectl apply -f` command:
+1. Deploy the following example `.yaml` file into the Kubernetes cluster by running the `kubectl apply -f` command:
```
Comment (hfxsd) [inline en/access-dashboard.md:51]: 请和开发确认下是否要把 `kubectl apply -f` 改成 `kubectl create -f`
本文有多处用到此命令，如需修改，请全文查找替换。

File: en/access-dashboard.md [hunk 8]
```diff
@@ -85,13 +91,11 @@ TiDB Dashboard is built in the PD component in TiDB 4.0 and later versions. You
     kubectl port-forward svc/basic-discovery -n ${namespace} 10262:10262
     ```
 
-3. Visit <http://localhost:10262/dashboard> in your browser to access TiDB Dashboard.
+    By default, `port-forward` binds to the IP address 127.0.0.1. If you need to use another IP address to access the machine running the `port-forward` command, you can add the `-address` option and specify the IP address to be bound.
```
Comment (hfxsd) [inline en/access-dashboard.md:94]:
```suggestion
    By default, `port-forward` binds to the IP address 127.0.0.1. If you need to use another IP address to access the machine running the `port-forward` command, you can add the `--address` option and specify the IP address to be bound.
```

File: en/access-dashboard.md [hunk 9]
```diff
@@ -85,13 +91,11 @@ TiDB Dashboard is built in the PD component in TiDB 4.0 and later versions. You
...
 
-> **Note:**
->
-> By default, `port-forward` binds to the IP address 127.0.0.1. If you need to use another IP address to access the machine running the `port-forward` command, you can add the `-address` option and specify the IP address to be bound.
+3. Visit <http://localhost:10262/dashboard> in your browser to access TiDB Dashboard.
 
-## Access TiDB Dashboard by Ingress
+## Method 2. Access TiDB Dashboard by Ingress
 
 In important production environments, it is recommended to expose the TiDB Dashboard service using Ingress.
```
Comment (hfxsd) [inline en/access-dashboard.md:100]:
```suggestion
In important production environments, it is recommended to expose the TiDB Dashboard service using `Ingress`.
```

File: zh/access-dashboard.md [hunk 10]
```diff
@@ -190,27 +194,12 @@ spec:
...
-
 ## TiDB Operator 中不支持的 Dashboard 功能
 
 TiDB Dashboard 中的部分功能会因为 kubernetes 的特殊环境而无法使用，包括以下功能：
 
-1. **概况** -> **监控和告警** -> **查看监控项**的链接无法正确跳转到 Grafana 监控页。如果需要访问 Grafana 监控，可以参考[访问 Grafana 监控面板](monitor-a-tidb-cluster.md#访问-grafana-监控面板)。
++ **概况** -> **监控和告警** -> **查看监控项**的链接无法正确跳转到 Grafana 监控页。如果需要访问 Grafana 监控，可以参考[访问 Grafana 监控面板](monitor-a-tidb-cluster.md#访问-grafana-监控面板)。
 
-2. 日志搜索功能无法使用。如果需要查看对应组件的日志，可以使用 `kubectl logs ${pod_name} -n {namespace}` 查看对应组件的日志或者通过 Kubernetes 集群的日志服务查看。
++ 日志搜索功能无法使用。如果需要查看对应组件的日志，可以使用 `kubectl logs ${pod_name} -n {namespace}` 查看对应组件的日志或者通过 Kubernetes 集群的日志服务查看。
```
Comment (hfxsd) [inline zh/access-dashboard.md:203]:
```suggestion
+ 日志搜索功能无法使用。如果需要查看对应组件的日志，可以使用 `kubectl logs ${pod_name} -n {namespace}` 查看对应组件的日志，或者通过 Kubernetes 集群的日志服务查看。
```

**Review by hfxsd — APPROVED** [review]
Rest LGTM

File: en/access-dashboard.md (hunk 7 above)
Comment (TomShawn) [reply en/access-dashboard.md:51]: 见 https://github.com/pingcap/docs-tidb-operator/pull/1507#discussion_r774398545，这里应该是可替换可不替换。

File: en/access-dashboard.md (hunk 9 above)
Comment (TomShawn) [reply en/access-dashboard.md:100]: 上下文都没加反引号

**Pull request merged**